src/py/flwr/superexec/exec_servicer.py

```diff
@@ -35,8 +35,9 @@
 class ExecServicer(exec_pb2_grpc.ExecServicer):
     """SuperExec API servicer."""
 
-    def __init__(self, executor: Executor) -> None:
+    def __init__(self, executor: Executor, config: Optional[Dict[str, str]]) -> None:
         self.executor = executor
+        self.executor.set_config(config)
```

Comment:
We shouldn't do this in the servicer, but instead pass an already-configured `executor` to the servicer